conjur/controller/init_controller.py

```diff
@@ -42,22 +42,30 @@ def load(self):
         """
         Method that facilitates all method calls in this class
         """
+        if self.conjurrc_data.conjur_url is None:
+            self.get_conjur_url()
```

Comment:
suggest renaming for `prompt_for_conjur_url()` the name `get_conjur_url` implies of a function that return the url ant not storing it inside the class variables

Reply:
fixed!